tools/check_python_version.py

```diff
@@ -0,0 +1,71 @@
+"""Check if the installed Python interpreter has correct version.
+
+This script has to be called with two command line arguments:
+expected_major_version expected_minor_version
+
+The script then check if actual Python version (major+minor) is
+the same or newer than expected version.
+
+Usage:
+python check_python_version.py 2.7
+python3 check_python_version.py 3.6
+python3 check_python_version.py 3.7
+etc.
+"""
+
+import sys
+
+
+def get_expected_version(arguments):
+    """Get the expected version from arguments provided on command line."""
+    if len(arguments) <= 2:
+        print("Usage python check_python_version.py major minor\n"
+              "      python3 check_python_version.py major minor")
+        raise Exception("CLI arguments missing")
```

Comment:
The exceptions handled and raised in python has to be specific rather than the top level class `Exception`. This comment goes for all the occurrences in this code.

Reply:
@samuzzal-choudhury actually I just reused this from stacks-report here: https://github.com/fabric8-analytics/f8a-stacks-report/blob/master/tools/check_python_version.py let me know if that works or should I change the exceptions to more specific ones?

Reply:
@samuzzal-choudhury @dipanjanS do we need it for the simple script isolated from other code and called from CLI? I agree it is a good approach for larger code (and yet we still use it in our repositories ...)

Reply:
For this case I think its just being used to check the python version and not used with the main codebase, so I'm good with it if you folks are :)